tests/api/api_routes/workflow/test_workflow.py

```diff
@@ -0,0 +1,113 @@
+from http import HTTPStatus
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from nestor_api.adapters.git.abstract_git_provider import AbstractGitProvider
+from nestor_api.api.flask_app import create_app
+
+
+@patch("nestor_api.api.api_routes.workflow.workflow.Logger", autospec=True)
+class TestWorkflow(TestCase):
+    def setUp(self):
+        app = create_app()
+        app.config["TESTING"] = True
+        self.app_client = app.test_client()
+
+    @patch("nestor_api.api.api_routes.workflow.workflow.get_git_provider", autospec=True)
+    @patch("nestor_api.api.api_routes.workflow.workflow.workflow_lib", autospec=True)
+    @patch("nestor_api.api.api_routes.workflow.workflow.config_lib", autospec=True)
+    def test_init_workflow(
+        self, config_mock, workflow_lib_mock, get_git_provider_mock, _logger_mock
+    ):
+        """Should properly return success response containing report."""
+        # Mock
+        fake_config = {"git": {"provider": "some-provider"}}
+        config_mock.get_project_config.return_value = fake_config
+        get_git_provider_mock.return_value = MagicMock(spec=AbstractGitProvider)
+        workflow_lib_mock.init_workflow.return_value = (
+            "success",
+            {"integration": {"created": (True, True), "protected": (True, True)}},
+        )
+
+        # Tests
+        response = self.app_client.post("/api/workflow/init/my-org/my-app")
+        data = response.get_json()
+
+        # Assertions
+        self.assertEqual(response.status_code, HTTPStatus.OK)
+        self.assertEqual(
+            data,
+            {
+                "organization": "my-org",
+                "message": "Workflow initialization succeeded",
+                "app": "my-app",
+                "report": {"integration": {"created": [True, True], "protected": [True, True]}},
```

Comment:
I don't really understand what the two booleans represent, would it make sense to simplify it this way ?
```suggestion
                "report": {"integration": {"created": True, "protected": True}},
```
Which means : "the integration branch has been created and protected"

Reply:
No. First stands for "Was it modified during the operation", second for the current value. 
Could not find a way to document it in a clearer way without a complete documentation